expression/generator/time_vec.go

```diff
@@ -609,6 +840,10 @@ func generateDotGo(fileName string) error {
 	if err != nil {
 		return err
 	}
+	err = subTime.Execute(w, subTimeSigsTmpl)
```

Comment:
I've noticed that the template of `subTime` is similar as `addTime`, could you unify them into one template?

We can define an template viriable to distinguish `add` or `sub`

```
{{ $addOrSub = slice .FuncName 0 3 }}
```

then use `$addOrSub` in every difference.

then exectute template like this.
```go
addOrSubTime.Execute(w, function{FuncName: "AddTime", Sigs: addDataSigsTmpl})
addOrSubTime.Execute(w, function{FuncName: "SubTime", Sigs: subDataSigsTmpl})
```

Reply:
@SilvaXiang Please address this comment.

Reply:
> @SilvaXiang Please address this comment.

OK , I will fix it this afternoon.